operator/tools/go.mod

```diff
@@ -3,10 +3,10 @@ module github.com/stackrox/stackrox/operator/tools
 go 1.19
 
 require (
-	github.com/kudobuilder/kuttl v0.15.0
+	github.com/kudobuilder/kuttl v0.14.0
```

Comment:
We actually need v0.15 because of https://github.com/kudobuilder/kuttl/pull/437

Reply:
But then I need to update `go: upgraded sigs.k8s.io/controller-tools v0.10.0 => v0.11.1` but I hope it's fine

Reply:
I hope so too :sweat_smile:
In fact I'm afraid changing to approach to a single go.mod has the potential to create much larger train wrecks given the current rate of breaking changes between k8s library API versions. But we'll see.